src/test/java/persistence/reflection/ReflectionTest.java

```diff
@@ -0,0 +1,107 @@
+package persistence.reflection;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.study.Car;
+import persistence.study.PrintView;
+
+import java.lang.reflect.*;
+import java.util.Arrays;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.*;
+
+
+public class ReflectionTest {
```

Comment:
```suggestion
class ReflectionTest {
```
junit 5부터는 테스트 클래스에 pulibc을 제거해주셔도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods